### DISCUSSION TITLE
Struggling with showing a page on Shellcontent

### DISCUSSION THREAD

**Discussion by pravgautam12** [dotnet/maui#15868] [Q&A]
Hello everyone,
Absolute beginner in .NET MAUI here. 
I have an AppShell.xaml file, that has shellcontents. It looks like this:

`<FlyoutItem FlyoutDisplayOptions="AsMultipleItems" Route= "ParentRoute">
        <ShellContent
            Title="Places"
            Icon="dotnet_bot.png"
            ContentTemplate="{DataTemplate local:YelpFusion}"
            Route="YelpFusion"
            />

        <ShellContent
            Title="For you"
            ContentTemplate="{DataTemplate local:MainPage}"
            Route="MainPage" 
            />

        <ShellContent
            Title="Newfound Gems"
            ContentTemplate="{DataTemplate local:MainPage}"
            Route="MainPage" />

        <ShellContent
            Title="Settings"
            ContentTemplate="{DataTemplate local:MainPage}"
            Route="MainPage"/>
    </FlyoutItem>`

Everything shows up well except the first Shell content whose DataTemplate is YelpFusion. All the other pages show MainPage perfectly. However, YelpFusion does not even appear in IntelliSense list if I start typing it. YelpFusion.xaml.cs file looks like this:

`public partial class YelpFusion : ContentPage
{
    public YelpFusion(double L, double G)
    {
        InitializeComponent();
        BindingContext = new YelpFusionViewModel(L, G);
    }
}`

The constructor of YelpFusionViewModel takes two parameters, L and G. There is a bunch of logic in that viewmodel which if it is important to share for me to get help, do let me know. 

TLDR: Cannot make YelpFusion page show in my Shellcontent, what could be the issue? 

Thank you <3
